packages/mobile/src/account/LinkBankAccountScreen.tsx

```diff
@@ -146,7 +156,30 @@ export function StepOne() {
     // Add a bit of a delay so that Persona can popup before switching to the spinner view
     setTimeout(() => setIsInPersonaFlow(true), 500)
   }
-  const onSuccessPersona = () => {
+
+  const isUserRegionSupported = (address: InquiryAttributes['address']) => {
+    if (!address) {
+      Logger.error(TAG, 'Address field does not exist in Persona inquiry attributes')
+      return false
+    }
+    if (!address.countryCode || address.countryCode !== 'US') {
+      Logger.info(TAG, 'User region country code not in US')
+      return false
+    }
+
+    if (!address.subdivisionAbbr || UNSUPPORTED_REGION_ABBR.includes(address.subdivisionAbbr)) {
+      // Finclusive currently do not support residents of NY and TX
+      Logger.info(TAG, 'User region state not supported by finclusive')
```

Comment:
same here, misleading if `address.subdivisionAbbr` is just falsy. In that case I'd expect to see a log like "missing field from address: `subdivisionAbbr`"